src/main/java/ymfc/ui/Ui.java

```diff
@@ -204,7 +204,22 @@ public void printDeletedTask(String deletedRecipe, int listCount) {
         System.out.println("\t  " + deletedRecipe);
         // Conditional operator to pluralize "recipe" when listCount above 1
         System.out.println("\tYou currently have " + listCount +
-                (listCount <= 1 ? " recipe" : " recipes") + " in your list.");
```

Comment:
What if there is no recipe left in the list? Should this line be alright if you leave it as it is? 🤔

Reply:
If there is no recipe left in the list, it will print:
`You currently have: 0 recipes in your list` which I believe is more grammatically correct

Reply:
Oh you're right! I've just searched this on Google! New knowledge for me! 👍